setup.py

```diff
@@ -41,5 +41,6 @@
         'License :: Public Domain',
         'Operating System :: OS Independent',
         'Topic :: Text Processing :: Markup'
-    ]
+    ],
+    zip_safe=False
```

Comment:
Could we relocate this to `include_package_data` and maybe add a comment? For example:

```python
# DEV: Include files like `VERSION` in our package
include_package_data=True,
# DEV: Mark our package as not zip safe so we can load `VERSION` on install
zip_safe=False,
install_requires=open('requirements.txt').readlines(),
# ...